### ISSUE TITLE
Incorrect Unirule mapping: UR001229799

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-annotation#3996]
UniProt UniRule (ID and label):
UniRule: UR001229799

Sequences with problematic annotation (ID + gene/protein name):
fly CG31337 (Q8SXS2)

Description of issue
UR001229799 adds several peptidase related entries based on the presence of Pfam signature PF12032. But that signature is a regulatory domain which is usually, but not always (as in Q8SXS2) associated with a peptidase domain.  Can the GO term 'GO:0004252 serine-type endopeptidase activity' and the keywords 'Hydrolase', 'Protease' and 'Serine protease' (which generate further GO annotations), at least, be removed from this UniRule please?

**Comment by PedroRaposo** [comment]
We looked into this rule and we determined that the annotation was inconsistent due to the Pfam signature regards to a domain, and not a family. We decided to add, to the rule, a new family signature that has the mentioned protease activity. with this no information was lost and we have a stronger annotation, and the protein Q8SXS2 will not be touched by the rule.
Is this change ok, in your view @sjm41 ?
